pkg/autodetect/main.go

```diff
@@ -62,21 +60,21 @@ func New(restConfig *rest.Config) (AutoDetect, error) {
 	}, nil
 }
 
-// Platform returns the detected platform this operator is running on. Possible values: Kubernetes, OpenShift.
-func (a *autoDetect) Platform() (platform.Platform, error) {
+// OSRoutesAvailability checks if OpenShift Routes is available.
```

Comment:
I think it should be:
```suggestion
// OSRoutesAvailability checks if OpenShift Routes are available.
```